docs/how-to-guides/create-backup.md

```diff
@@ -10,6 +10,10 @@ including the **durability files**: snapshots and WALs that contain Memgraph's
 data in a recoverable format. On startup, it searches for previously saved
 durability files and uses them to recreate the most recent DB state.
 
+When talking about the data directory, we are actually talking about two
+directories, `snapshots` and `wal` which are usually located in the
+`/var/lib/memgraph` folder.
+
```

Comment:
I think in general the data directory is the directory that is specified as the `--data-directory`, which contains other directories, such as `snapshots`, `wal`, but other directories also, `streams`, `triggers`, etc.

Reply:
i can add
```suggestion
When talking about the data directory in the context of backup and restore, we are actually talking about two
directories, `snapshots` and `wal` which are usually located in the
`/var/lib/memgraph` folder.

```

Reply:
@antaljanosbenjamin what do you think about this change?

Reply:
Perfect!